### ISSUE TITLE
Text in language selection dropdown shows up as white on a white background

### ISSUE THREAD

**Issue by CSumm** [ifmeorg/ifme#1162]
# Description 
Text in language selection dropdown shows up as white on a white background.

# Reproduction Steps

Enter If-Me.org.
Scroll down to footer where language dropdown is located.
Open language dropdown by clicking on it.

# Screenshots
![accessibilityissue_language dropdown](https://user-images.githubusercontent.com/31711543/46921317-e576a680-cfc7-11e8-803e-d0b6f36aa3e3.jpg)

# Do you want to be the assignee to work on this?

✅

**Comment by CSumm** [comment]
This seems to occur only on a PC. Tried on Ubuntu and didn't receive this error.

**Comment by gpetrioli** [comment]
The real issue here is that `option` element does not support opacity. Now, the `select` element is styled with opacity (*it is white but with and alpha channel of 0.1 out of 1*) but the `option` discard the alpha channel and show as white. 

We would need to define a specific color, but stylistically it cannot match the `select` because, as it is semi-transparent, it is defined by the background image in the footer.

We could use media queries to approximate a color based on what is shown behind the select box but we can never be exact.

Let me know if you want me to proceed with this and if the proposed solution is adequate.

**Comment by julianguyen** [comment]
@gpetrioli You're spot on 👌 @CSumm started a [PR](https://github.com/ifmeorg/ifme/pull/1167) for this. It would be great if you could help review it and provide suggestions.

**Comment by CSumm** [comment]
I think his explanation of how to fix it seems like the answer. We could work together on this issue if you would like.

**Comment by CSumm** [comment]
I think working on the PR together would be best. The PR I wrote was rather sloppy and was close to irreversibly bad.

**Comment by gpetrioli** [comment]
Ok, after some crossbrowser testing i see that most browsers do not show transparent options (*when the select box is open*) but they are just not affected because they also do not change the color of the `option` elements. 

So just setting `color: initial` fixes the problem everywhere.
